test/controllers/SecuritiesControllerSpec.scala

```diff
@@ -140,16 +138,16 @@ class SecuritiesControllerSpec extends SpecBase {
   "statementsUnavailablePage" should {
 
     "render correctly" in {
-      val app: Application = application().build()
-      val appConfig        = app.injector.instanceOf[AppConfig]
-      val unavailableView  = app.injector.instanceOf[security_statements_not_available]
+      val app = applicationBuilder().build()
```

Comment:
There's a val called `application` in spec base that does this. Can we use that instead?

Reply:
I used that first but its creating conflicts (sometimes at run time) between application created in the Setup and the common Application instance (retrieved from application) due to their start and shutdown states, for these controllers. So better to get the application instance this way (using common applicationBuilder) in order to avoid the conflict for these controllers.